src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs

```diff
@@ -126,4 +126,55 @@ public static string Format(string resourceFormat, object p1, object p2, object
             return string.Format(resourceFormat, p1, p2, p3);
         }
     }
+
+    internal partial class StringsHelper
+    {
+        internal class ResourceNames
+        {
+            internal const string DataCategory_Advanced = @"Advanced";
```

Comment:
Do we want to fix the remaining types in a separate PR?
cc @cheenamalhotra

Reply:
NetFx converts all Strings.resx strings to constant. We can do that all, if needed. I do not believe all of it is needed.

Reply:
I have added the components to SqlCommand and SqlConnection as well.